website/docs/d/is_backup_policy_jobs.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_backup_policy_jobs"
+description: |-
+  Get information about BackupPolicyJobCollection
+---
+
+# ibm_is_backup_policy_jobs
+
+Provides a read-only data source for BackupPolicyJobCollection. For more information, about backup policy in your IBM Cloud VPC, see [Backup policy jobs](https://cloud.ibm.com/docs/vpc?topic=vpc-backup-view-policy-jobs).
+
+**Note:** 
+VPC infrastructure services are a regional specific based endpoint, by default targets to `us-south`. Please make sure to target right region in the provider block as shown in the `provider.tf` file, if VPC service is created in region other than `us-south`.
+
+**provider.tf**
+
+```terraform
+provider "ibm" {
+  region = "eu-gb"
+}
+```
+
+## Example Usage
+
+```terraform
+data "ibm_is_backup_policy_jobs" "example" {
+  backup_policy_id = ibm_is_backup_policy.example.id
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+- `backup_policy_id` - (Required, String) The backup policy identifier.
+- `target_snapshot_crn` - (Optional, String) Filters the collection to resources with the target snapshot with the specified CRN.
```

Comment:
Filters are not working, please verify.

Reply:
it's not array
target_snapshot = []
source_volume = []